policyengine_us/parameters/gov/states/wa/tax/income/credits/working_families_tax_credit/min_amount.yaml

```diff
@@ -1,11 +1,10 @@
 description: Washington Working Families Tax Credit minimum amount for those with nonzero benefit.
 metadata:
-  label: WA Working Families Tax Credit minimum amount
-  name: wa_wftc_min_amount
+  label: Washington Working Families Tax Credit minimum amount
   unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.08.0206 Exemptions—Working families—Eligible low-income persons—Penalties. (3)(c)"
+    - title: RCW 82.08.0206 Exemptions—Working families—Eligible low-income persons—Penalties. (3)(c)
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.08.0206
 values:
   2022-01-01: 50
```

Comment:
no tax form reference?

Reply:
didn't find any
seems that they use website to fill tax form